CoinDock-app/src/App/Api/walletapi.js

```diff
@@ -0,0 +1,56 @@
+import { getUserId } from "App/Auth/helper";
+import baseApi from "./api";
+
+const wallet = baseApi.injectEndpoints({
+  endpoints: (build) => ({
+    addWallet: build.mutation({
+      query: ({
+        coin,
+        walletname,
+        wallet_id,
+
+        ...data
+      }) => ({
+        url: `/v1/users/${getUserId()}/add-wallet`,
+        method: "post",
+        data: {
+          coin: coin,
+          name: walletname,
+          wallet_id: wallet_id,
+        },
+      }),
+      invalidatesTags: [
+        "linechart",
+        "filter",
+        "coinfilter",
+        "coinshortname",
+        "pie",
+        "piefilter",
+        "total",
+        "primarycurrency",
+        "topperformer",
+        "lowperformer",
+        "coincard",
+      ],
+      transformResponse: (response) => {
```

Comment:
I think transform response is not needed here

Reply:
removed